js/coinflex.js

```diff
@@ -0,0 +1,2354 @@
+        //          "timestamp": "1651619029297",
+        //          "code": "710003",
+        //          "message": "FAILED sanity bound check as price (4.000) >  upper bound (3.439)",
+        //          "clientOrderId": "1651619024219000",
+        //          "price": "4.000",
+        //          "quantity": "3.0",
+        //          "side": "BUY",
+        //          "marketCode": "BAND-USD",
+        //          "timeInForce": "GTC",
+        //          "orderType": "LIMIT"
+        //     }
+        //
+        const data = this.safeValue (response, 'data', []);
+        const firstOrder = this.safeValue (data, 0, {});
+        this.checkOrderResponseForException (response, firstOrder);
+        return this.parseOrder (firstOrder, market);
+    }
+
+    async editOrder (id, symbol, type, side, amount = undefined, price = undefined, params = {}) {
+        const market = this.market (symbol);
+        const [ request, query ] = await this.buildOrderRequest (market, type, side, amount, price, params);
+        request['orders'][0]['orderId'] = id;
+        const response = await this.privatePostV2OrdersModify (this.extend (request, query));
+        //
+        //     {
+        //         "event": "modifyOrder",
+        //         "timestamp": "1651695117070",
+        //         "accountId": "38422",
+        //         "data": [
+        //             {
+        //                 "success": "true",
+        //                 "timestamp": "1651696843859",
+        //                 "clientOrderId": "1651692616185",
+        //                 "orderId": "1002128500938",
+        //                 "price": "0.5600",
+        //                 "quantity": "8.0",
+        //                 "side": "BUY",
+        //                 "marketCode": "XRP-USD",
+        //                 "timeInForce": "GTC",
+        //                 "notice": "OrderModified",
+        //                 "orderType": "LIMIT"
+        //             }
+        //         ]
+        //     }
+        //
+        //
+        // Note, for inexistent order-id, the order-object might be like:
+        //
+        //             {
+        //                 "success": "false",
+        //                 "timestamp": "1651695117070",
+        //                 "code": "40035",
+        //                 "message": "Open order not found with clientOrderId or orderId",
+        //                 "price": "0.56",
+        //                 "quantity": "8",
+        //                 "side": "BUY",
+        //                 "marketCode": "XRP-USD",
+        //                 "orderType": "LIMIT"
+        //             }
+        //
+        const data = this.safeValue (response, 'data', []);
+        const firstOrder = this.safeValue (data, 0, {});
+        this.checkOrderResponseForException (response, firstOrder);
+        return this.parseOrder (firstOrder, market);
+    }
+
+    async cancelOrders (ids, symbol = undefined, params = {}) {
```

Comment:
Can't we add the singular version of this method (cancelOrder)? It could be probably just a proxy to this one. 

Example
```Javacript
async cancelOrder (id, symbol = undefined, params = {}) {
    const ids = [ id ];
    return await this.cancelOrders(ids, symbol, params);
}
```

Reply:
oh, right. seems i had added 'true' and forgot to add the actual method. Btw, good idea to have that defined in base, like `fetchTicker` works in base. I'll go PR that in base.

Reply:
added cancelOrder inside coinFlex, as in base it doesnt seem to be appropriate to have emulated cancelOrder though cancelOrders.